static/compiler.js

```diff
@@ -385,6 +385,12 @@ Compiler.prototype.getEffectiveFilters = function () {
     if (filters.execute && !this.compiler.supportsExecute) {
         delete filters.execute;
     }
+    var disabledFilters = this.compiler.disabledFilters;
+    _.each(disabledFilters, function (filter) {
+        if (filters[filter] && disabledFilters.indexOf(filter) !== -1) {
```

Comment:
I'm a bit confused here... you're iterating over disabledFilters anyway. How can the `indexOf` check here ever fail? Or am I totally confused by this code? :)